pkgs/tools/admin/rset/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchurl, coreutils, openssh, gnutar }:
+
+stdenv.mkDerivation rec {
+  pname = "rset";
+  version = "2.1";
+
+  src = fetchurl {
+    url = "https://scriptedconfiguration.org/code/${pname}-${version}.tar.gz";
+    sha256 = "0916f96afl8kcn2hpj4qhg92g2j93ycp2sb94nsz3q44sqc6ddhb";
+  };
+
+  patches = [ ./paths.patch ];
+
+  postPatch = ''
+    substituteInPlace rset.c \
+      --replace @ssh@       ${openssh}/bin/ssh \
+      --replace @miniquark@ $out/bin/miniquark \
+      --replace @rinstall@  $out/bin/rinstall \
+      --replace @rsub@      $out/bin/rsub
```

Comment:
Can we also just replace the bare binary names or does this cause issues?

Reply:
It would be easy to over-replace, and there are cases like this where we need to patch anyway:

https://github.com/NixOS/nixpkgs/pull/120595/files#diff-00160293b5e7505f1d8a5f2bfd51a9d22acaceb39e66e785996f4342e4e3fdcdR104

I think, at that point, might as well patch the other binary names so they can be safely, selectively replaced.